src/file.rs

```diff
@@ -35,10 +37,23 @@ where
     let dyn_data = volume.into_ndarray::<T>().unwrap();
     let data = dyn_data
         .into_dimensionality::<Ix3>()
-        .expect("Loaded NIfTI image should be a 3D array");
+        .expect("Loaded NIfTI image must be a 3D array");
     (header, data)
 }
 
+/// Creates a TrackVis file reader for further data mapping.
+pub fn fibers_reader<P: AsRef<Path>>(path: P, nifti_header: &NiftiHeader) -> Reader {
+    let path = path.as_ref();
+    if !path.exists() {
+        panic!("Fibers {path:?} doesn't exist.");
+    }
+    let dim = &nifti_header.pixdim;
```

Comment:
Can you please name it `pixdim` or `spacing`? `dim` sounds like the image dimension.

Reply:
The reason I use shorter names sometimes is to fight `rustfmt`, I like to keep my code on fewer lines when possible if it helps with clarity. `pixdim` works fine though :)
```rust
Reader::new(path)
    .expect("The path should exists")
    .to_voxel_space(Vector3::new(pixdim[1], pixdim[2], pixdim[3]))
```